### PR TITLE
OPM: child registration fix

### PR DESCRIPTION
**Pull request by lwyszomi**
@czue @esoergel 

last comments in case: http://manage.dimagi.com/default.asp?187634

### CODE REVIEW

**Comment by esoergel** [comment]
Test failure looks real, likely just needs to be updated
```
======================================================================
FAIL: test_service_unavailable_at_all (custom.opm.tests.test_child_condition_four.TestChildBirthRegistration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/opm/tests/test_child_condition_four.py", line 96, in test_service_unavailable_at_all
    self.assertEqual(True, self.get_condition(row))
AssertionError: True != False
```

**Comment by czue** [comment]
still failing (on a different test). maybe run them locally so you don't have to wait for travis?
```
======================================================================
FAIL: test_service_unavailable_different_condition (custom.opm.tests.test_child_condition_four.TestChildBirthRegistration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/opm/tests/test_child_condition_four.py", line 131, in test_service_unavailable_different_condition
    self.assertEqual(True, self.get_condition(row))
AssertionError: True != False
----------------------------------------------------------------------
```

**Comment by lwyszomi** [comment]
@czue 
Sure I run them locally. I thought that the travis shows me the all failing tests.

**Comment by czue** [comment]
ah, no it uses --failfast

**Comment by lwyszomi** [comment]
@czue now I know :)

**Comment by lwyszomi** [comment]
@czue @esoergel 
I fixed tests, I removed the ```ServiceAvailabilityTestMixIn``` from ```TestChildBirthRegistration``` because currently in method ```child_birth_registered``` we don't check service.

File: custom/opm/tests/test_child_condition_four.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def setUp(self):
         self.condition_getter = lambda row: row.child_received_measles_vaccine
 
 
-class TestChildBirthRegistration(OPMCaseReportTestBase, ConditionFourTestMixin, ServiceAvailabilityTestMixIn):
+class TestChildBirthRegistration(OPMCaseReportTestBase, ConditionFourTestMixin):
```
Comment (esoergel) [inline custom/opm/tests/test_child_condition_four.py]: :star2:

**Pull request merged**